cmd/docker/builder.go

```diff
@@ -84,10 +80,10 @@ func processBuilder(dockerCli command.Cli, cmd *cobra.Command, args, osargs []st
 		return args, osargs, nil, nil
 	}
 
-	if useLegacy {
+	if buildKitDisabled {
 		// display warning if not wcow and continue
 		if dockerCli.ServerInfo().OSType != "windows" {
-			_, _ = fmt.Fprintln(dockerCli.Err(), newBuilderError(true, nil))
+			_, _ = fmt.Fprintf(dockerCli.Err(), "%s\n\n", buildkitDisabledWarning)
```

Comment:
... so I added the extra newlines back here when _printing_ (if returned as an error, we don't have the trailing newline.